packages/usa-memorable-date/src/styles/_usa-memorable-date.scss

```diff
@@ -37,3 +37,7 @@
 .usa-form-group--year {
   width: units(9);
 }
+
+.usa-form-group--month {
+  width: units(card-lg);
+}
```

Comment:
Is this necessary? It's a modest breaking change and I'm  not sure it adds anything since a select in the existing layout should expand to fit its content

Reply:
Pretty sure this can go away and we don't have a potential breaking layout change